document/core/binary/modules.rst

```diff
@@ -479,9 +507,11 @@ The lengths of vectors produced by the (possibly empty) :ref:`function <binary-f
      \Bcustomsec^\ast \\ &&&
      \elem^\ast{:\,}\Belemsec \\ &&&
      \Bcustomsec^\ast \\ &&&
+     \X{dc}^?{:\,}\Bdatacountsec \\ &&&
+     \Bcustomsec^\ast \\ &&&
      \X{code}^n{:\,}\Bcodesec \\ &&&
      \Bcustomsec^\ast \\ &&&
-     \data^\ast{:\,}\Bdatasec \\ &&&
+     \data^{\X{dc'}}{:\,}\Bdatasec \\ &&&
```

Comment:
Nit: use metavariable *m*, analogous to *n* for func/code section

Reply:
Done.